tm2/pkg/bft/consensus/replay_test.go

```diff
@@ -1144,13 +1152,11 @@ func TestHandshakeUpdatesValidators(t *testing.T) {
 	genDoc, _ := sm.MakeGenesisDocFromFile(genesisFile)
 	handshaker := NewHandshaker(stateDB, state, store, genDoc)
 	proxyApp := appconn.NewAppConns(clientCreator)
-	if err := proxyApp.Start(); err != nil {
-		t.Fatalf("Error starting proxy app connections: %v", err)
-	}
-	defer proxyApp.Stop()
-	if err := handshaker.Handshake(proxyApp); err != nil {
-		t.Fatalf("Error on abci handshake: %v", err)
-	}
+	err := proxyApp.Start()
+	require.NoError(t, err, "Error starting proxy app connections")
+	defer func() { err := proxyApp.Stop(); require.NoError(t, err) }()
```

Comment:
This error check can be inlined

Reply:
done in 926e226f7944da16f18cf760413b2fdb1cc560ca